src/Stratis.Bitcoin/Utilities/DBreezeSerializer.cs

```diff
@@ -11,11 +11,14 @@ namespace Stratis.Bitcoin.Utilities
     /// </summary>
     public class DBreezeSerializer
     {
+        public NetworkOptions NetworkOptions { get; private set; }
+
         /// <summary>
         /// Initializes custom serializers for DBreeze engine.
         /// </summary>
-        public void Initialize()
+        public void Initialize(NetworkOptions networkOptions = null)
```

Comment:
Ah here we can defo make `NetworkOptions` mandatory.

Now I am happy @Aprogiena made it an instance (and not static)

Reply:
Added TODO to make NetworkOptions required